lib/plugins/basalprofile.js

```diff
@@ -101,19 +101,38 @@ function init (ctx) {
   };
 
   function basalMessage(slots, sbx) {
-    var basalValue = sbx.data.profile.getTempBasal(sbx.time);
-    var response = 'Unable to determine current basal';
-    var preamble = '';
-    if (basalValue.treatment) {
-      preamble = (slots && slots.pwd && slots.pwd.value) ? slots.pwd.value + ' has a ' : 'Your ';
-      var minutesLeft = moment(basalValue.treatment.endmills).from(moment(sbx.time));
-      response = preamble + 'temp basal of ' + basalValue.totalbasal + ' units per hour will end ' + minutesLeft;
-    } else {
-      preamble = (slots && slots.pwd && slots.pwd.value) ? slots.pwd.value + ' ' : 'Your ';
-      response = preamble + 'current basal is ' + basalValue.totalbasal + ' units per hour';
+        var basalValue = sbx.data.profile.getTempBasal(sbx.time);
+        var response = 'Unable to determine current basal';
+        var preamble = '';
+        if (basalValue.treatment) {
+            preamble = (slots && slots.pwd && slots.pwd.value) ? translate('alexaPreamble3person', {
```

Comment:
Will these work even in the non-Alexa use-case?

Reply:
Do you know where else this is used? I checked the unit tests, they are still intact. I didn't check anything else as I don't know where this might be used.